tests/test_config.py

```diff
@@ -1,32 +1,50 @@
 import os
 
+potential_paths = [
+    'tests/',
+    '../tests/'
+    'tap-square/tests/',
+    '../tap-square/tests/',
+]
+
+def go_to_tests_directory():
+    for path in potential_paths:
+        if os.path.exists(path):
+            os.chdir(path)
+            return os.getcwd()
+    raise NotImplementedError("This check cannot run from {}".format(os.getcwd()))
+
+##########################################################################
+### TEST
+##########################################################################
 
-cwd = os.getcwd()
-if cwd not in {'root/project', '/opt/code/tap-square'}:
-    print("WARN: This script is meant to run from the top level directory of the tap.")
+print("Acquiring path to tests directory.")
+cwd = go_to_tests_directory()
 
 print("Reading in filenames from tests directory.")
-print("Parsing directory for tests.")
-files = [
-    name for name in os.listdir(cwd + '/tests')
-    if 'test' == name[:4] and '.py' == name[-3:] and \
-    name not in ['test_client.py', 'test_config.py']
-]
-print("\tFiles found: {}".format(files))
+files_in_dir = [name for name in os.listdir(cwd)]
+
+print("Dropping files that are not of the form 'test_<feature>.py'.")
+test_files_in_dir = [fn for fn in files_in_dir if 'test' == fn[:4] and '.py' == fn[-3:]]
```

Comment:
You say in the print "test_" but you only check for "test" prefix. Next can you use string startswith and endswith for the strings instead of the `[:4]` and `[-3:]` because I feel they're easier to read?

Reply:
Done